pyrogram/utils.py

```diff
@@ -53,8 +53,7 @@ def get_input_media_from_file_id(
     file_type = decoded.file_type
 
     if expected_file_type is not None and file_type != expected_file_type:
-        raise ValueError(
-            f"Expected {expected_file_type.name}, got {file_type.name} file id instead")
+        raise ValueError(f"Expected {expected_file_type.name}, got {file_type.name} file id instead")
```

Comment:
What's the code width the project follows?

Reply:
120 characters